socorro/siglists/README.md

```diff
@@ -0,0 +1,3 @@
+# Signatures Utilities Lists
+
+Great documentation comes here.
```

Comment:
I actually sticked to markdown because, well, I like it more than rst, and we use rst in the docs only. I don't think it matters much. :)